Dockerfile

```
@@ -21,7 +21,7 @@ RUN        mkdir -p /data/{warc,cdxj,ipfs}
 
 # Download and install IPFS
 ENV        IPFS_PATH=/data/ipfs
-ARG        IPFS_VERSION=v0.12.2
+ARG        IPFS_VERSION=v0.13
```

```suggestion
ARG        IPFS_VERSION=v0.13.0
```

Partial versions don't work in this context. In the `ibnesayeed/setup-ipfs` Action I have implemented a custom solution to resolve semver. :)